qsiprep/workflows/anatomical/volume.py

```diff
@@ -169,34 +169,61 @@ def init_anat_preproc_wf(
 
     dwi_only = config.workflow.anat_modality == "none"
 
-    # Make sure we have usable anatomical reference images/masks
-    get_template_image = pe.Node(
+    # XXX: This is a temporary solution until QSIPrep supports flexible output spaces.
+    get_template = pe.Node(
         GetTemplate(
             template_name=config.workflow.anatomical_template,
-            infant_mode=config.workflow.infant,
             anatomical_contrast=config.workflow.anat_modality,
         ),
         name="get_template_image",
     )
+    mask_template = pe.Node(
+        afni.Calc(expr="a*b", outputtype="NIFTI_GZ"),
```

Comment:
So this is moved from the interface into the workflow? That makes a lot more sense